01 - Community/Video Channels/🗂️ Video Channels.md

```diff
@@ -9,12 +9,3 @@ publish: true
 # 🗂️ Video Channels
 
 #placeholder/description 
-
```

Comment:
I would delete this file, the manual MOC can then be the one for YouTube channels

Reply:
OK. Links to this: 
![image](https://user-images.githubusercontent.com/4840096/139856081-1a3f34f2-3821-47a3-863f-f0ed6b1c0346.png)